src/EventListener/AssetsInstallCommandListener.php

```diff
@@ -37,8 +37,6 @@ final class AssetsInstallCommandListener
     public const METHOD_ABSOLUTE_SYMLINK = 'absolute symlink';
     public const METHOD_RELATIVE_SYMLINK = 'relative symlink';
 
-    protected static $defaultName = 'assets:install';
```

Comment:
This is use to run the command `bin/console assets:install` so it's needed.

Reply:
Oh, it's not a command

Reply:
I thought the same first

Reply:
It is extension for command. This line can be deleted. IMO better will be use it in line 71 insted.
```php
if ('assets:install' !== $command->getName() || null !== $coreBundle) {
    return;
}
```